openproblems/tasks/dimensional_reduction/README.md

```diff
@@ -5,7 +5,48 @@ Dimensional reduction is one of the key challenges in single-cell data represent
 
 Each *feature* in a dataset functions as a single dimension. In the AnnData structure, these are stored in the `adata.var` component. Each cell analyzed is stored in `adata.obs`. The measured cells are thereby represented in the **cell x gene expression matrix**, `adata.X`.
 
-![AnnData Structure](/images/core_anndata.png)
+![AnnData Structure](./images/core_anndata.png)
 
 While each of the ~30,000 dimensions measured in each cell (not accounting for roughly 75-90% data dropout per cell, another issue entirely), likely contribute to some sort of data structure, the overall structure of the data is diluted due to the <a href = "https://en.wikipedia.org/wiki/Curse_of_dimensionality">*curse of dimensionality*</a>. In short, it's difficult to visualize the contribution of each individual gene in a way that makes sense to the human eye, i.e., two or three dimensions (at most). Thus, we need to find a way to <a href = "https://en.wikipedia.org/wiki/Dimensionality_reduction">*dimensionally reduce*</a> the data for visualization and interpretation.
 
+Here, we add `adata.obsm`, which contains coordinates for various dimensional reduction methods. UMAP results are stored as `adata.obsm['X_umap']`, *t*-sne results as `adata.obsm['X_tsne']` and PCA results as `adata.obsm['X_pca']`.
+
+### CITE-seq dataset
```

Comment:
The README is mostly for developers to understand how to add new datasets/methods/metrics. This doesn't seem like important information.